### ISSUE TITLE
Could  you  give  some   instructions abou  how to use it?

### ISSUE THREAD

**Issue by feixiangdekaka** [varunagrawal/tiny-faces-pytorch#3]

**Comment by varunagrawal** [comment]
I'm planning to write full docs for this next week. Currently super busy with some deadlines. Sorry and thank you for your patience!

**Comment by feixiangdekaka** [comment]
> I'm planning to write full docs for this next week. Currently super busy with some deadlines. Sorry and thank you for your patience!

I  have  some  bug :+1: 
1. crop_image  (self,img,bboxes)     return three result ,but  main function  receive  4,crop_box have  no result.


2.widerface.py     ann   is  not defined,  there is   something  wrong.

3.when input size =500  the res3's output is 63  ,however the res4 is  64  for resnet101   ,when excute  score = score_res3 + score4  make error .  dimensions is  not  equal   63not  equal 64.

**Comment by VladVin** [comment]
Just upload the weights 😀

**Comment by varunagrawal** [comment]
@feixiangdekaka thanks for reporting the bugs. I have fixed all of them in today's refactor.
Also, the README should now have everything you need to run this. If it works for you, please let me know, so I can close this issue.

@VladVin yup, need to retrain the models and make sure the accuracy is the same as the paper.

**Comment by feixiangdekaka** [comment]
> @feixiangdekaka thanks for reporting the bugs. I have fixed all of them in today's refactor.
> Also, the README should now have everything you need to run this. If it works for you, please let me know, so I can close this issue.
> 
> @VladVin yup, need to retrain the models and make sure the accuracy is the same as the paper.

WIP   ???   waiting in  precess?haha

**Comment by varunagrawal** [comment]
Um Work In Progress. I have to retrain the models 😅

**Comment by arki0207** [comment]
I have a bug.
wider_face.py can't handle this situation：  a train image  has no bbx .
![image](https://user-images.githubusercontent.com/6756108/56197022-8b37ca80-606a-11e9-8c62-b9ac7386428e.png)

**Comment by varunagrawal** [comment]
@arki0207 can you please try with the latest code? It should be able to handle it.

**Comment by varunagrawal** [comment]
@arki0207 that line makes no sense. If the number of bounding boxes is 0, why should the dataset have a line `0 0 0 0 0 0 0 0 0 0`? It should be blank.

I can't understand why you would encounter this issue in the first place.

**Comment by varunagrawal** [comment]
Closing this issue since the original issue has been fixed. Please open new issues for new problems.
